### PR TITLE
WalletNotFound error in listEosWallets fixed.

### PR DESCRIPTION
**Pull request by denisshevchenko**
<p align="right">#354</p>

# Overview

- [x] I have changed `"/wallets/externally-owned"` path in API to `"externally-owned-wallets"`.
- [x] I have changed corresponding integration tests.
- [x] I have removed `pending` for `EOSWALLETS_LIST_01`.

# Comments

All integration tests are passed now, including `EOSWALLETS_LIST_01`.

### CODE REVIEW

File: src/Cardano/Wallet/API/V1/Wallets.hs [hunk 1]
```diff
@@ -43,33 +43,28 @@ type FullyOwnedAPI = Tag "Wallets" 'NoTagDescription :>
     )
 
 type ExternallyOwnedAPI = Tag "Externally Owned Wallets" 'NoTagDescription
-    :> ( "wallets"
-        :> "externally-owned"
+    :> ( "externally-owned-wallets"
```
Comment (piotr-iohk) [inline src/Cardano/Wallet/API/V1/Wallets.hs:46]: don't want to be picky, but maybe something shorter would be more convenient for usage, like `eo-wallets`? :)

Comment (KtorZ) [reply src/Cardano/Wallet/API/V1/Wallets.hs:46]: I'd avoid acronym in APIs. Having more descriptive names also serve a documentation purpose :+1:

**Comment by KtorZ** [comment]
~~@denisshevchenko May you look into updating the Servant layout golden test?~~

Just saw your last commit :)

**Comment by piotr-iohk** [comment]
@denisshevchenko, is there anything extra that needs to be changed? Because I see the API doc still shows `/api/v1/wallets/externally-owned` -> https://input-output-hk.github.io/cardano-wallet/#tag/Externally-Owned-Wallets (I thought CI would update the API doc)

**Comment by denisshevchenko** [comment]
@piotr-iohk Documentation will be updated automatically by CI.

**Comment by piotr-iohk** [comment]
lgtm :+1: 
api doc will be updated by nightly CI run on develop.

**Review by piotr-iohk — APPROVED** [review]

**Comment by KtorZ** [comment]
@piotr-iohk Not nightly, it happens after every merge into develop :+1:

**Comment by piotr-iohk** [comment]
ah, cool :)

**Pull request merged**